Sofa/Component/Topology/Container/Dynamic/src/sofa/component/topology/container/dynamic/EdgeSetGeometryAlgorithms.inl

```diff
@@ -453,69 +437,15 @@ bool is_point_on_edge(const Vec& p, const Vec& a, const Vec& b)
         return false;
 }
 
-template<class DataTypes>
-auto EdgeSetGeometryAlgorithms<DataTypes>::computeRest2PointsBarycoefs(
-    const sofa::type::Vec<3, Real>& p,
-    PointID ind_p1,
-    PointID ind_p2) const -> sofa::type::vector<SReal>
-{
-    constexpr Real ZERO = static_cast<Real>(1e-6);
-
-    sofa::type::vector< SReal > baryCoefs;
-
-    const typename DataTypes::VecCoord& vect_c = (this->object->read(core::ConstVecCoordId::restPosition())->getValue());
-    const typename DataTypes::Coord& c0 = vect_c[ind_p1];
-    const typename DataTypes::Coord& c1 = vect_c[ind_p2];
-
-    sofa::type::Vec<3,Real> a; DataTypes::get(a[0], a[1], a[2], c0);
-    sofa::type::Vec<3,Real> b; DataTypes::get(b[0], b[1], b[2], c1);
-
-    Real dis = (b - a).norm();
-    Real coef_a, coef_b;
-
-
-    if(dis < ZERO)
-    {
-        coef_a = 0.5;
-        coef_b = 0.5;
-    }
-    else
-    {
-        coef_a = (p - b).norm() / dis;
-        coef_b = (p - a).norm() / dis;
-    }
-
-    baryCoefs.push_back(coef_a);
-    baryCoefs.push_back(coef_b);
-
-    return baryCoefs;
-
-}
-
 template<class Vec>
 sofa::type::vector< typename Vec::value_type > compute_2points_barycoefs(const Vec& p, const Vec& a, const Vec& b)
```

Comment:
What is the use of this method ? Isn't it exactly the same as the compute2PointsBarycoefs except from the return type ? 
The fact that it does the same as the other one and its name makes me want to remove it...

Reply:
ok for me, I will depreciate it as well

Reply:
ah in fact it is not even a method of the class... bye bye